src/components/ModalEnterPhone.tsx

```diff
@@ -0,0 +1,97 @@
+import { connected } from 'process';
+import React,  {FC, useEffect, useState} from 'react';
+import './ModalEnterPhone.scss';
+
```

Comment:
You should use completed component PasswordAndConfirm

Reply:
What it mean?